### PR TITLE
Merge release-1.0.0 into main

### PR DESCRIPTION
**Pull request by andyleveto**
This repo is under change control. All changes must be made in a branch from main and reviewed.

## Description of Changes

Various minor changes required for MuleSoft certification. New readme/license files, naming convention, POM dependency changes.

## Security

**REMINDER: All file contents are public.**

[x] I have ensured no secure credentials or sensitive information remain in code, metadata, comments, etc. Of particular note:
No temporary testing changes committed such as API base URLs, access tokens, print/log statements, etc.
[x] My changes do not introduce any security risks, or any such risks have been properly mitigated.

Describe briefly what security risks you considered, why they don't apply, or how they've been mitigated.

## Reviewers

[x] I have assigned the appropriate reviewer(s). 

Minimally, a second set of eyes is needed ensure no non-public information is published. Consider also including subject-matter experts and privacy/security reviewers, if appropriate.

### CODE REVIEW

File: doc/release-notes.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Orchestrate Convert API Release Notes
```
Comment (lynnfaraday) [inline doc/release-notes.md]: Remove this file in favor of the GitHub releases functionality.

File: pom.xml [hunk 2]
```diff
@@ -5,16 +5,16 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <name>Orchestrate Convert API</name>
-    <description>The Convert API transforms your input data between standardized formats (FHIR, HL7, C-CDA, and more) and enhances the output through terminology normalization, de-duplication, and combining FHIR bundles to create a consolidated single person record. https://orchestrate.docs.careevolution.com/convert/</description>
+    <name>Orchestrate Convert API Connector - Mule 4</name>
```
Comment (lynnfaraday) [inline pom.xml:8]: Is this Mule 4 thing required by them?

**Review by lynnfaraday — COMMENTED** [review]
Minor comments. I think we should get Jeff (or whomever) to test this out, since I can't really comment on whether it works just by looking at the java.

File: doc/release-notes.md (hunk 1 above)
Comment (andyleveto) [reply doc/release-notes.md]: Done

File: pom.xml (hunk 2 above)
Comment (andyleveto) [reply pom.xml:8]: Yes, this is required here

**Comment by jsovel** [comment]
I've done functional testing by calling the API published on the Mulesoft Exchange using the Mulesoft Anypoint Studio, and the testing was successful.

**Review by jsovel — APPROVED** [review]
Performed functional testing with the published connector.  It works as expected.

**Pull request merged**